packages/terra-notification-dialog/src/terra-dev-site/doc/example/ContentNotificationDialog.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Button from 'terra-button';
+import NotificationDialog, { NotificationDialogVariants } from '../../../NotificationDialog';
+
+const clickConfirm = () => {
+  console.log('You clicked confirm'); // eslint-disable-line no-console
+};
+
+class ContentNotificationDialog extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    return (
+      <div>
+        <NotificationDialog
+          variant={NotificationDialogVariants.ALERT}
+          isOpen={this.state.isOpen}
+          title="Make sure that the title relates directly to the choices."
+          startMessage="The Main Instruction is text used to provide more detail or define terminology. Don’t repeat the title verbatim."
+          content={(
```

Comment:
Maybe use a hyperlink for the content, I think that's a common use case that this is intended to meet.

Reply:
Double checking with @neilpfeiffer, I was under the impression the List was also a common use.

Reply:
A list is fine, I was just thinking that the example looked a little disjointed. I'm sure @neilpfeiffer will have better input.